dev-python/pyside/metadata.xml

```diff
@@ -6,17 +6,26 @@
 		<name>Gentoo Qt Project</name>
 	</maintainer>
 	<use>
-		<flag name="X">Build QtGui and QtTest modules</flag>
+		<flag name="concurrent">Build QtConcurrent module</flag>
 		<flag name="declarative">Build QtDeclarative module</flag>
 		<flag name="designer">Build QtDesigner and QtUiTools modules</flag>
+		<flag name="gui">Build QtGui and QtTest modules</flag>
```

Comment:
**Wow.** That is indeed a typo, presumably inherited from the prior `pyside-9999.ebuild`. Your attention to detail is impressive. I who am about to submit a working pull request salute you!